packages/devtools-launchpad/src/index.js

```diff
@@ -159,7 +164,7 @@ async function bootstrap(React, ReactDOM, App, appActions, appStore) {
 
   const { store, actions, LaunchpadApp } = await initApp();
   renderRoot(React, ReactDOM, LaunchpadApp, store);
-  await getTabs(actions);
+  await connectClients(actions);
```

Comment:
If I read this correctly, every 3 seconds we empty the tab list and fetch the firefox tabs.
Isn't it a problem when debugging Chrome/Node?